src/commands/entities/convert.ts

```diff
@@ -30,11 +30,12 @@ entity type, you will have to provide additional information as
 explained below.
 
 Regex entities make use of regular expressions specific to a single
-locale. Use the --pattern flag to provide the regular expression
-for the converted entity. The regular expression provided gets
-applied to all locales in the project. Use the entities:configure
-command to update the regular expression on a per locale basis
-if needed.
+locale. Use the --pattern flag to provide the regular expression for
+the converted entity. It is recommended to surround the pattern value
+with quotes, especially if the "\\" is used in the pattern. The regular
+expression provided gets applied to all locales in the project. Use the
```

Comment:
"The regular expression provided gets applied to all locales in the project." -- can you explain the effects of this more?